NeosModLoader/ModConfiguration.cs

```diff
@@ -527,7 +527,7 @@ internal void Save(bool saveDefaultValues = false, bool immediate = false)
 		{
 
 			Thread thread = Thread.CurrentThread;
-			NeosMod? callee = Util.ExecutingMod();
+			NeosMod? callee = Util.ExecutingMod(new(1));
```

Comment:
`new(1)` is a stack trace? 0.o .NET syntax is weird, I assume this has just omitted the `StackTrace` part and it's implied...?

Reply:
Yes, the compiler knows a StackTrace is the only type that can go there, so we can omit the explicit type in the constructor call.